utils/configen/mkconfig.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+
+	"github.com/BurntSushi/toml"
+	"github.com/coniks-sys/coniks-go/keyserver"
```

Comment:
I'm not sure if this util should belong in `keyserver/util` since, I also use it to generate bot configurations. https://github.com/coniks-sys/coniks-go/blob/aed779393105af0d4658c389f097574e40373d6e/utils/configen/mkconfig.go

Should I make it 2 separate utils?